app/_kong_plugins/jwt-signer/examples/disable-signing-tokens.yaml

```
@@ -0,0 +1,20 @@
+description: Disables signing or resigning access and channel tokens.
+
+extended_description: |
+  Disables signing or resigning access and channel tokens. 
+  This can be useful when the token is just validated at Kong and doesn't need to be passed to the upstream.
+min_version: 3.12 
+title: 'Disable signing or resigning tokens'
+
+weight: 900
+
+config:
+  channel_token_signing: false
+  access_token_signing: false
+
+tools:
+  - deck
+  - admin-api
+  - konnect-api
+  - kic
+  - terraform
```

```suggestion
tools:
  - deck
  - admin-api
  - kic
```
This plugin isn't available in konnect (and the terraform instructions are for konnect terraform).